### PR TITLE
Graph drilldown panel

### LINKED ISSUE #250: Clean up drill-down panes in the Graphs space to make them look closer to the Models one
`NodePane.vue` 
image

`EdgePane.vue` 
image

For the pieces of evidence containers, we can get some inspiration from Causemos
image

### PR DESCRIPTION
**Pull request by YohannParis**
**What**
- based off the CSS clean up branch #255 , so it might be better to merge 255 first.
- fix #250 

**How**
- Tidy up the code and CSS
- Added COSMOS call on the evidence to display the Title, Authors, and Year of publication
- Created a Loading Alert component to show that information are being fetch

**Screenshot Node panel**

Screen Shot 2021-07-08 at 14 54 06

**Video Evidence panel**

https://user-images.githubusercontent.com/636801/124975448-0a26ce00-dffc-11eb-85d2-ecbae74b9b8c.mov

### CODE REVIEW

File: client/src/views/Graphs/components/DrilldownPanel/EdgePane.vue [hunk 1]
```diff
@@ -57,40 +76,73 @@
...
 <style scoped>
-  ul {
-    list-style-type: none;
-    margin-left: 0;
-    padding-left: 0;
-  }
-
-  li {
+  figure {
     border: var(--border);
```
Comment (RosaRomeroGomez) [inline client/src/views/Graphs/components/DrilldownPanel/EdgePane.vue:126]: What about if the `--border` variable is something more subtle: 
`  --border: 1px solid #f6f6f6;
`

File: client/src/views/Graphs/components/DrilldownPanel/EdgePane.vue [hunk 2]
```diff
@@ -57,40 +76,73 @@
...
     cursor: pointer;
-    padding: 4px 8px;
+    font-size: .9em;
+    margin: .5em 0;
+    padding: .5rem;
+  }
+
+  figure:hover {
+    border-color: var(--selection);
```
Comment (RosaRomeroGomez) [inline client/src/views/Graphs/components/DrilldownPanel/EdgePane.vue:134]: I think the yellow doesn't work that great with the white background, not enough contrast. This should be probably `--selection-dark`.

File: client/src/components/widgets/LoadingAlert.vue [hunk 3]
```diff
@@ -0,0 +1,37 @@
+<template>
+  <span class="alert alert-info">
+    Loading more information
```
Comment (RosaRomeroGomez) [inline client/src/components/widgets/LoadingAlert.vue:3]: Maybe we could use the `fa-spinner` icon here? https://fontawesome.com/v4.7/icon/spinner

Comment (YohannParis) [reply client/src/components/widgets/LoadingAlert.vue:4]: https://user-images.githubusercontent.com/636801/125091810-c20cb780-e09e-11eb-9ba2-5b48450f1e1f.mov

File: client/src/views/Graphs/components/DrilldownPanel/EdgePane.vue (hunk 1 above)
Comment (YohannParis) [reply client/src/views/Graphs/components/DrilldownPanel/EdgePane.vue]: Screen Shot 2021-07-09 at 10 18 49

File: client/src/views/Graphs/components/DrilldownPanel/EdgePane.vue (hunk 2 above)
Comment (YohannParis) [reply client/src/views/Graphs/components/DrilldownPanel/EdgePane.vue]: Screen Shot 2021-07-09 at 10 18 49

**Review by RosaRomeroGomez — APPROVED** [review]
Thanks!

**Pull request merged**